static/style.css

```diff
@@ -58,18 +52,22 @@ body{
 }
 
 .CathoLogo {
-    align-self: center;
+    display: flex; 
+    align-items: center;
+
 }
 
 .VisibleMenu {
     color: #BCBF12;
     font-weight: bold;
     display: flex;
-    justify-content: space-between;
+    justify-content: flex-end;
```

Comment:
This case is related to the use case of the class commenting above. justify-content: flex-end; it is only necessary when the class is used in nav. When used in Hide, this attribute is unnecessary. This demonstrates inconsistent use of the class for various purposes and can be difficult to maintain.